backend/src/backend/logging.clj

```diff
@@ -27,12 +27,26 @@
   ([from to]
    (j/query db [(str "SELECT * FROM system_log WHERE DATE(created) between '" from "' and '" to "'")])))
 
+(defn get-device-from-id
+  [device_id]
+  (j/query db [(str "SELECT FROM device WHERE device_id ='" device_id "'")]))
```

Comment:
We should be using the argument syntax for queries here and not just string templating them ourselves.